docs/docs/self-host/self-host-lightdash.mdx

```diff
@@ -0,0 +1,94 @@
+# Self-Host Lightdash
+
+Lightdash is MIT licensed and open source. You can self-host Lightdash on your own infrastructure.
+
+This guide is designed for DevOps Engineers that are familiar with Docker, Kubernetes, and are comfortable 
+configuring environment variables, SMTP credentials, and database connections.
+If you're unsure whether to self-host please read our guide on [Lightdash Cloud vs. Self-Hosted](./lightdash-cloud-vs-self-hosted.mdx).
+
+## Prerequisites for self-hosting
+
+ - Access to a kubernetes cluster and [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed
+ - [Helm](https://helm.sh/docs/intro/quickstart/)
+ - Docker
```

Comment:
Also `sops`

Reply:
Only if using `helm secrets` but I don't mention that in the docs here, I'll leave secrets management to the user.